wpeverest/everest-forms
# Fix - Email Setting Glitches, Add email setting issue

### All Submissions:






### Changes proposed in this Pull Request:



Email setting had a conflict when adding another email, first email setting gets changed. Also there was show hide issue on pdf section.
This issue has been fixed in this PR.

### How to test the changes in this Pull Request:

1. Create Form
2. Add multiple email in the setting tabs
3. Verify if its working, and no glitches on the UI.

### Types of changes:




### Other information:




### Changelog entry
> Fix - Email Setting Glitches, Add email setting issue
